data/ch5/ch5_load.sh

```diff
@@ -1,29 +1,46 @@
 #!/bin/bash
 
-cd /path/to/dse-6.8.0_20190911-LABS/dsbulk-1.3.4/bin
-
-echo "Loading vertices into the graph neighborhoods_prod"
-
-./dsbulk load -k neighborhoods_prod -t Transaction -url /path/to/graph-book/data/ch5/Transactions.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_prod -t Vendor -url /path/to/graph-book/data/ch5/Vendors.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_prod -t Customer -url /path/to/graph-book/data/ch5/Customers.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_prod -t Loan -url /path/to/graph-book/data/ch5/Loans.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_prod -t Account -url /path/to/graph-book/data/ch5/Accounts.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_prod -t CreditCard -url /path/to/graph-book/data/ch5/CreditCards.csv -header true -schema.allowMissingFields true
-
-echo "Completed loading vertices into the graph neighborhoods_prod."
-
-echo "Loading edges into the graph neighborhoods_prod"
-
-./dsbulk load -k neighborhoods_prod -t Customer__owes__Loan -url /path/to/graph-book/data/ch5/owes.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_prod -t Customer__owns__Account -url /path/to/graph-book/data/ch5/owns.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_prod -t Customer__uses__CreditCard -url /path/to/graph-book/data/ch5/uses.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_prod -t Transaction__charge__CreditCard -url /path/to/graph-book/data/ch5/charge.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_prod -t Transaction__deposit_to__Account -url /path/to/graph-book/data/ch5/deposit_to.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_prod -t Transaction__withdraw_from__Account -url /path/to/graph-book/data/ch5/withdraw_from.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_prod -t Transaction__pay__Loan -url /path/to/graph-book/data/ch5/pay_loan.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_prod -t Transaction__pay__Vendor -url /path/to/graph-book/data/ch5/pay_vendor.csv -header true -schema.allowMissingFields true
-
-echo "Completed loading edges into the graph neighborhoods_prod."
-
-cd /path/to/graph-book/data/ch5
+# customizable variables
+DSBULK_PATH=/path/to/dse-6.8.0_20190911-LABS/dsbulk-1.3.4
+DEST_KS=neighborhoods_prod
+
+# 
+OLDWD="`pwd`"
+SCRIPTIR="`dirname $0`"
+cd "$SCRIPTDIR"
+DATADIR="`pwd`"
+
+DSBULK="`which dsbulk`"
+if [ -z "$DSBULK" ]; then
+    if [ ! -f "$DSBULK_PATH/bin/dsbulk" ]; then
+        echo "Please set DSBULK_PATH variable to top-level path of DSBulk distribution"
+        echo "  or add directory with dsbulk to PATH, like, 'PATH=...dsbulk-1.3.4/bin:\$PATH'"
+        exit 1
+    fi
```

Comment:
We need to add `DSBULK=$DSBULK_PATH/bin/dsbulk` After fi.

Reply:
@alexott thank you! this resolves the issue.

I have cherry-picked this commit to the top of this branch. Then, I refactored all other loading processes to reflect the suggestions. I will open a PR into master for you to review.

Reply:
@denisekgosnell I'll review your PR tomorrow, I have some suggestions for your open PR, similar to the https://github.com/datastax/graph-book/pull/3, but I need to make similar changes in the all scripts. If you can look to my PR, and it looks ok for you, I can make changes in all scripts